src/axom/CMakeLists.txt

```diff
@@ -76,7 +76,8 @@ endif()
 
 # If Axom is the main project, set FOLDER property for top-level check targets
 if ("${PROJECT_SOURCE_DIR}" STREQUAL "${CMAKE_SOURCE_DIR}")
-    set(_code_check_targets "docs" "doxygen_docs"
+    #set(_code_check_targets "docs" "doxygen_docs"
+    set(_code_check_targets "docs"
```

Comment:
@nselliott , I forget whether this was intentional or if it got in by mistake.

Reply:
Hmmm, this must be a mistake.  Unrelated to anything we did here.  I'll clean it up.